src/directives/user-menu/user-menu.directive.scss

```diff
@@ -4,41 +4,37 @@
   cursor: pointer;
 
   &.expanded {
-    color: white;
+    color: var(--gw-button-primary-active-color);
 
     .user-menu-toggle {
-      background: var(--primary-color);
-
-      &:hover {
-        background: var(--primary-color);
-      }
+      background: var(--gw-button-primary-active-background);
+      border-color: var(--gw-button-primary-active-background);
 
       .dropdown-icon {
         transform: rotate(180deg);
       }
 
       .user-icon {
-        color: white;
+        color: inherit;
       }
     }
   }
 
   .user-menu-toggle {
     padding: 8px 12px;
-    background: var(--base-dropdown-color);
+    background: var(--gw-button-primary-background);
     display: flex;
     align-items: center;
     max-width: 20rem;
     gap: 6px;
     cursor: pointer;
+    border: solid var(--gw-button-primary-background);
+    border-top-left-radius: 8px;
+    border-bottom-left-radius: 8px;
```

Comment:
Maybe we can extract a variable for the border radius. I see it is `8px` almost everywhere. Although, I don't expect further restyling of this UI ...

Feel free to ignore this one.